TKeazirian.HTTPServer/EchoServer.cs

```diff
@@ -19,16 +19,18 @@ public static void StartListening()
             listener.Bind(endPoint);
             listener.Listen(10);
 
-            Console.WriteLine("Waiting for a connection...");
-
             while (true)
             {
                 var handler = listener.Accept();
 
                 _request = GetRequest(handler);
 
                 var responseToSend = Controller.GenerateResponse(_request);
-                handler.Send(responseToSend);
+
+                handler.NoDelay = true;
+
+                handler.Send(responseToSend, SocketFlags.None);
+                handler.Close();
```

Comment:
Is `handler` a good variable name? A handler is something that typically runs in response to an event.  What is the triggering event here?